packages/vuetify/src/util/globals.ts

```diff
@@ -0,0 +1,23 @@
+function checkForEventListenerPassiveSupport (supported = false): boolean {
+  try {
+    const opts = Object.defineProperty({}, 'passive', {
+      get: () => (supported = true),
+    })
+
+    window.addEventListener('check', opts, opts)
+    window.removeEventListener('check', opts, opts)
+  } catch (e) {
+    console.warn('The current browser does not support the passive option.')
+    console.warn(e)
```

Comment:
Why?

Reply:
Elaborate

Reply:
This warning is pointless, passive is just a performance thing.